docs/devices/E1744.md

```diff
@@ -103,11 +103,17 @@ It's not possible to read (`/get`) or write (`/set`) this value.
 The minimal value is `0` and the maximum value is `100`.
 The unit of this value is `%`.
 
+### Brightness (numeric)
+Current Brightness value as an integer between 0-255.
+Value can be found in the published state on the `brightness` property.
+It's not possible to read (`/get`) or write (`/set`) this value.
+The minimal value is `0` and the maximum value is `255`.
+
 ### Action (enum)
 Triggered action (e.g. a button click).
 Value can be found in the published state on the `action` property.
 It's not possible to read (`/get`) or write (`/set`) this value.
-The possible values are: `brightness_move_up`, `brightness_move_down`, `brightness_stop`, `toggle`, `brightness_step_up`, `brightness_step_down`.
+The possible values are: `rotate_left`, `rotate_right`, `rotate_stop`, `play_pause`, `skip_backward`, `skip_forward`.
```

Comment:
These are the legacy actions, the listed actions will appear when you set `legacy: false` for this device.